index.html

```diff
@@ -48,7 +48,7 @@
           </nav>
         </header>
         <section class="heading-section">
-          <div>
+          <div class="hero-content">
```

Comment:
- [ ] Please make sure that the popup is responsive
![image](https://user-images.githubusercontent.com/74744787/180625083-956fc6ca-425d-4b7c-a349-8b0081e4ea4f.png)
!! the buttons are not appearing in the mobile  version